frontend/javascripts/test/reducers/flycam_reducer.spec.ts

```diff
@@ -152,3 +155,47 @@ test("Flycam should move by plane in ortho mode with dynamicSpaceDirection", (t)
   );
   equalWithEpsilon(t, getPosition(newState.flycam), [0, 0, -2]);
 });
+test("Flycam should not change additional coordinates value when layers don't have any.", (t) => {
+  const newState = FlycamReducer(
+    initialState,
+    FlycamActions.setAdditionalCoordinatesAction([{ name: "t", value: 123 }]),
+  );
+  t.deepEqual(initialState, newState);
+});
+
+test("Flycam should get correct subset of additional coordinates value when subset is set.", (t) => {
+  const adaptedState = update(initialState, {
+    // flycam
+    dataset: {
+      dataSource: {
+        dataLayers: {
+          $set: [
+            {
+              name: "color1",
+              type: "color",
+              additionalCoordinates: [{ name: "t", bounds: [0, 10] }],
+            },
+            {
+              name: "color2",
+              type: "color",
+              additionalCoordinates: [{ name: "u", bounds: [10, 20] }],
+            },
+          ],
+        },
+      },
+    },
+  });
+  const newState = FlycamReducer(
+    adaptedState,
+    // t is passed, but u is missing. Instead, a superfluous
+    // q is passed.
+    FlycamActions.setAdditionalCoordinatesAction([
+      { name: "t", value: 123 },
+      { name: "q", value: 0 },
+    ]),
+  );
+  t.deepEqual(newState.flycam.additionalCoordinates, [
+    { name: "t", value: 123 },
```

Comment:
Is it relevant that t is now 123 but the bounds of "t" are actually only 0 to 10 according to the layer configuration?

Reply:
Good point, I adapted the test so that both cases (valid and invalid range) are covered.